package.json

```diff
@@ -15,14 +15,15 @@
   },
   "homepage": "https://github.com/dwyl/abase#readme",
   "devDependencies": {
+    "eslint": "^3.5.0",
     "istanbul": "^0.4.5",
     "pre-commit": "^1.1.3",
     "tape": "^4.6.0"
   },
   "scripts": {
-    "test": "tape ./test/*.js",
-    "coverage": "istanbul cover tape ./test/*.js && istanbul check-coverage --statements 100 --functions 100 --lines 100 --branches 100",
-    "eslint": "eslint ."
+    "test": "node_modules/.bin/tape ./src/test/*.js",
+    "eslint": "node_modules/.bin/eslint .",
+    "coverage": "node_modules/.bin/istanbul cover node_modules/.bin/tape ./src/test/*.js"
```

Comment:
you've duplicated the test command, how about "node_modules/.bin/istanbul cover npm run test"